utils/tcpros_utils.js

```diff
@@ -245,6 +245,21 @@ let TcprosUtils = {
     return null;
   },
 
+  validateServiceClientHeader(header, service, md5sum) {
+    if (!header.hasOwnProperty('service')) {
+      return 'Connection header missing expected field [service]';
+    }
+    else if (!header.hasOwnProperty('md5sum')) {
+      return 'Connection header missing expected field [md5sum]';
+    }
+    else if (header.service !== service) {
+      return 'Got incorrect service [' + header.service + '] expected [' + service + ']';
```

Comment:
In ES6, you can compose the string inline:  `Got incorrect service [${header.service}] expected [${service}]`